api/v1beta1/kogitoinfra_types.go

```diff
@@ -121,15 +122,9 @@ func (k *KogitoInfraStatus) GetVolumes() []api.KogitoInfraVolumeInterface {
 	return volumes
 }
 
-// SetVolumes ...
-func (k *KogitoInfraStatus) SetVolumes(infraVolumes []api.KogitoInfraVolumeInterface) {
-	var volumes []KogitoInfraVolume
-	for _, volume := range infraVolumes {
-		if newVolume, ok := volume.(KogitoInfraVolume); ok {
-			volumes = append(volumes, newVolume)
-		}
-	}
-	k.Volumes = volumes
+// AddVolumes ...
+func (k *KogitoInfraStatus) AddVolumes(infraVolume api.KogitoInfraVolumeInterface) {
```

Comment:
Same as above

Reply:
reverted back this function to `SetVolumes()`